src/js/apps/patients/patients-main_app.js

```diff
@@ -84,9 +89,14 @@ export default RouterApp.extend({
       'done-last-thirty-days': 'lastThirty',
     };
 
-    if (!worklistsById[worklistId]) Radio.trigger('event-router', 'notFound');
+    if (!worklistsById[worklistId] || !worklistType) Radio.trigger('event-router', 'notFound');
```

Comment:
should there be a route that doesn't return !worklistType?  I could see the type being wrong.. which maybe that should 404.

Reply:
This is paired with the `worklistNotFound` event above. Basically, it's accounting for a url like `worklist/owned-by-me`, but now I'm thinking that's overkill. I could remove that logic and add a check for `worklistType` equaling either `flows` or `actions`.

Reply:
Yeah I just think a route like `worklistNotFound` is never needed.